pkg/plugins/client_test.go

```diff
@@ -168,18 +169,15 @@ func TestNewClientWithTimeout(t *testing.T) {
 	m := Manifest{[]string{"VolumeDriver", "NetworkDriver"}}
 
 	mux.HandleFunc("/Test.Echo", func(w http.ResponseWriter, r *http.Request) {
-		time.Sleep(time.Duration(600) * time.Millisecond)
+		time.Sleep(time.Duration(20) * time.Millisecond)
```

Comment:
nit: The cast isn't needed:
```suggestion
		time.Sleep(20 * time.Millisecond)
```